test/utils.sh

```diff
@@ -487,3 +487,41 @@ function replace_value_in_yaml_file() {
   OLDVAL=$1; NEWVAL=$2; FILE=$3
   sed -i'.bak' -e "s#$OLDVAL#$NEWVAL#g" "$FILE"
 }
+
+function verify_deployment() {
```

Comment:
I'd suggest renaming this to `verify_carts_deployment` or `verify_sockshop_deployment`, as it is very specific to that project

Reply:
good point - i have changed the name of the function accordingly